README.md

```diff
@@ -1,12 +1,32 @@
-# my-component
+# Storage Writer
 
 [![Build Status](https://travis-ci.org/keboola/my-component.svg?branch=master)](https://travis-ci.org/keboola/my-component)
 
-> Fill in description
+Writes tables from input mapping to the configured bucket in the destination project.
```

Comment:
Popsat jak připravit bucket a token? dokud nebude ui.

Reply:
koukám že je tam teda configuration schema, ale pro vývoj by to bylo fajn mít v readme.